build.gradle

```diff
@@ -237,6 +237,21 @@ allprojects {
                     force "com.google.errorprone:error_prone_annotations:${googleErrorProneAnnotationsVersion}"
                     // Force patched version of GPRC, dependency of a number of Google service APIs in WNPRC and fileTransfer
                     force "io.grpc:grpc-context:${grpcVersion}"
+
+                    // TODO Remove these when the Picard version is upgraded to use a version >=1.57.2
```

Comment:
Please give some pointer to the module this comes from (SequenceAnalysis) as it makes it easier to track down whether a version has been updated (or removed).